indico/modules/events/timetable/client/js/reducers.ts

```diff
@@ -131,6 +131,11 @@ export default {
         } else {
           newEntries[newDayKey] = layout(
             newEntries[newDayKey].map(e => {
+              if (entry.sessionBlockId === e.id && e.type === EntryType.SessionBlock) {
+                e.children = e.children.map(child => {
+                  return child.id === entry.id && child.type === entryType ? entry : child;
```

Comment:
Not a blocking comment but I wonder if we should have some kind of util to see if an two entries are equal.

Also, shouldn't this be solved already just with `id` as it is a string that contains the type and id (eg. `c22`)